pkg/csi_driver/node_test.go

```diff
@@ -458,3 +459,84 @@ func TestNodePublishVolumeWILabelCheck(t *testing.T) {
 		}
 	}
 }
+
+func TestRemoveDisallowedMountOptions(t *testing.T) {
```

Comment:
Just in case, also add nil check somewhere / throw error if disallowed DisallowedMountOptions is passed nil map